index.html

```diff
@@ -294,53 +294,89 @@ <h1 class="title">About
+                                fill="currentColor">
+                                <path fill-rule="evenodd"
+                                    d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                                    clip-rule="evenodd" />
+                            </svg>
                         </div>
-                    </div>
-                </div>
-                <div class="language-section">
-                    <div class="dropdown-menu border-b">
-                        <p class="lang-text">Frameworks</p>
-                        <svg xmlns="http://www.w3.org/2000/svg" class="icon"
-                        viewBox="0 0 20 20" fill="currentColor">
-                        <path fill-rule="evenodd"
-                            d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
-                            clip-rule="evenodd" />
-                    </svg>
-                    </div>
-                </div>
-                <div class="language-section">
-                    <div class="dropdown-menu border-b">
-                        <p class="lang-text">Skills</p>
-                        <svg xmlns="http://www.w3.org/2000/svg" class="icon"
-                        viewBox="0 0 20 20" fill="currentColor">
-                        <path fill-rule="evenodd"
-                            d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
-                            clip-rule="evenodd" />
-                    </svg>
-                    </div>
-                </div>
+                    </li>
+                </ul>
             </div>
         </section>
+        <footer class="contact-section-bg">
+            <div class="contact-section">
+                <h1 class="title">Contact me</h1>
+                <p class="details">If you have an application you are interested in developing, a feature that you need
+                    built or a
+                    project that needs coding. I’d love to help with it </p>
+                <form action="https://formspree.io/f/mjvjzbal" method="POST" class="form">
+                    <ul>
+                        <li class="form-field">
+                            <input placeholder="Name" class="input" type="text" minlength="1" maxlength="30" name="name"
+                                required>
+                        </li>
+                        <li class="form-field">
+                            <input placeholder="Email" class="input" type="email" name="email" required>
+                        </li>
+                        <li class="form-field">
+                            <textarea placeholder="Write your message here" class="text-area" name="message" rows="10"
+                                maxlength="500"></textarea>
```

Comment:
- [ ] Please add `required` attribute for `textarea` too as you did above because the form is submitting even if the `textarea` is empty.

Reply:
thanks 😄